app/app/settings.py

```diff
@@ -31,6 +31,15 @@
 RATELIMIT_USE_CACHE = 'default'
 RATELIMIT_VIEW = 'tdi.views.ratelimited'
 
+# Gitcoinbot Account Information
+GITHUB_API_USER = os.environ.get('GITHUB_API_USER')
+GITHUB_API_TOKEN = os.environ.get('GITHUB_API_TOKEN')
+
+# Gitcoinbot APP Information
+GITCOINBOT_APP_ID =  os.environ.get('GITCOINBOT_APP_ID')
+github_pem_file = open("gitcoinbot-test.2017-12-27.private-key.pem", 'r')
```

Comment:
This can be bad news.  If you absolutely have to open a file, please use a context manager here if possible to ensure the file is always closed appropriately.